pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -103,6 +137,7 @@ func (z *zedrouter) getNIPortConfig(
 		if port == nil {
 			continue
 		}
+		basicNTPServerIPs, _ := types.GetNTPServers(*z.deviceNetworkStatus, port.IfName)
```

Comment:
I think it would be less confusing if getNIPortConfig didn't set NTPServers at all (not even the already resolved IPs) and would leave this to attachNTPServersToPortConfigs.
Then also please add a comment for getNIPortConfig mentioning that NTP servers are set separately.

Reply:
done